Tests/BotTests/Canned Data/DispatchServiceQueueStates.swift

```diff
@@ -89,3 +89,21 @@ let DispatchServiceQueueStates: Data = """
   }
 ]
 """.data(using: .utf8)!
+
+
+let DispatchServiceQueueStatesString = """
+## Merge Queue for target branch: branch1 ##
+
+State(
+ - status: integrating PR #1 (some-branch),
+ - queue:\(" ")
```

Comment:
had to use that `\(" ")` otherwise Xcode kept removing that trailing space from the string literal 🤦‍♂  (which made the test constantly fail, despite me trying to re-add the space and re-saving the file)